.github/ISSUE_TEMPLATE/kubernetes_update.md

```diff
@@ -46,12 +46,21 @@ List of items to do for upgrading to {version_1} {version_2} {version_3}
     - [ ] Regenerate allocated API endpoints: [make gen-allocation-grpc](https://github.com/googleforgames/agones/blob/main/build/README.md#make-gen-allocation-grpc)
 - [ ] Confirm the update works as expected by running e2e tests
     - [ ] Add the new supported Kubernetes versions to the e2e clusters creation
-        - [ ] In `terraform/e2e/module.tf`, add the new supported version to the map `kubernetes_versions`. Noted the location of the new clusters should have enough quota (CPU, In-use IP addresses) to create the cluster. And the new supported version is usually only available in RAPID channel.
+        - [ ] In `build/terraform/e2e/module.tf`, add the new supported version to the map `kubernetes_versions`. Noted the location of the new clusters should have enough quota (CPU, In-use IP addresses) to create the cluster. And the new supported version is usually only available in RAPID channel.
         - [ ] Recreate clusters with new scripts: `cd build; make GCP_PROJECT=agones-images gcloud-e2e-test-cluster`
     - [ ] Update the Cloud Build configuration to run e2e test on the new created clusters, and disable the e2e test on the cluster with the oldest supported K8s version
         - [ ] Update the `versionsAndRegions` variable to add the new supported version and remove the oldest supported K8s version in `cloudbuild.yaml` `submit-e2e-test-cloud-build` step
         - [ ] Run `make lint` for code quality check.
         - [ ] Submit a PR to trigger the e2e tests and verfiy they all pass
     - [ ] After the PR that includes the above Cloud Build configuration change has been merged and all the existing pending PRs in the Cloud Build queue have picked up the new configuration, submit a separate PR to update the e2e clusters terraform module to remove the e2e cluster with the oldest supported K8s version.
-        - [ ] In `terraform/e2e/module.tf`, remove the oldest supported version from the map `kubernetes_versions`.
+        - [ ] In `build/terraform/e2e/module.tf`, remove the oldest supported version from the map `kubernetes_versions`.
         - [ ] Destroy the old clusters with new scripts: `cd build; make GCP_PROJECT=agones-images gcloud-e2e-test-cluster`
```

Comment:
If I understand correctly, the `make GCP_PROJECT=agones-images gcloud-e2e-test-cluster` command didn't remove the cluster last time, so we used `terraform init -backend-config="bucket=agones-images-e2e-infra-bucket-tfstate" -backend-config="prefix=terraform/state"
terraform apply -var project="agones-images"` command. Was this an intermittent issue, or do we need to update it?

Reply:
I have a fix for that. https://github.com/googleforgames/agones/pull/3468 still pending review. No need to update the docs here.

Reply:
Cool, all good!